frontend/package.json

```diff
@@ -42,6 +43,7 @@
     "primeflex": "^3.3.1",
     "primeicons": "^6.0.1",
     "primevue": "^3.32.0",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "vue-i18n": "^9.5.0"
```

Comment:
Do you need to specify `vue-i18n` here explicitly, or it it sufficient to let `@nuxtjs/i18n` pick the version implicitly? I ask because specifying overly specific dependencies can cause versioning issues, so if this is just a transient for us, might as well drop it.

Reply:
Yep - we directly import from it in some of the code already :)